docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-vpc.md

```diff
@@ -22,7 +22,7 @@ To view VPC details, select a VPC in the list to display the **VPC Details** she
 
 To terminate a VPC, click the **Delete** icon for the VPC in the list you want to terminate, and then click **Terminate**. You can also terminate a VPC by clicking **Terminate VPC** in the **VPC Details** sheet. You can't terminate a VPC with active peering connections or clusters.
 
-## Create a VPC
+## Prerequisites
```

Comment:
In our cloud docs, these sections are called "Prerequisites" or "Before you start". For consistency, I suggest picking one and sticking with it (the platform docs have "Prerequisites").